modules/tfm/tfm/boards/CMakeLists.txt

```diff
@@ -179,3 +179,8 @@ if(LOG_MEMORY_PROTECTION)
   src/log_memory_protection.c
   )
 endif()
+
+target_link_options(tfm_s
+  PRIVATE
+    $<$<C_COMPILER_ID:GNU>:-Wl,--print-memory-usage>
+)
```

Comment:
Strange.

Why isn't this working:

ncs/modules/tee/tf-m/trusted-firmware-m/toolchain_GNUARM.cmake
61:    set(MEMORY_USAGE_FLAG LINKER:--print-memory-usage)

?